src/database/postgres/create_tables.md

```diff
@@ -0,0 +1,31 @@
+## Create tables in a Postgres database
+
+[![postgres-badge]][postgres] [![cat-database-badge]][cat-database]
+
+Use the `postgres` crate to create tables in a Postgres database. 
+
+The recipe assumes an existing database named `library`, the username is `postgres` and the password is `postgres`.
```

Comment:
Please include a description of the Connection::connect method, and a link to the api: https://docs.rs/postgres/*/postgres/struct.Connection.html#method.connect